packages/babel-plugin-syntax-top-level-await/.npmignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+*.log
+src
```

Comment:
nit: should also ignore `test`

Reply:
I just copy-pasted this file from somewhere else in the repo; we should fix many of them.